### ISSUE TITLE
confusing error message when LibScout finds no APKs in a dir

### ISSUE THREAD

**Issue by eighthave** [reddr/LibScout#14]
When running _LibScout_ on a directory that has no APKs, it gives an error that made me think that I had wrong command line flags:

```console
$ mkdir /tmp/newdir
$ java -jar build/LibScout.jar -o match -a lib/android-23.jar -p profiles  /tmp/newdir/
 
Command line parsing failed:
You have to provide a path to a single application file or a directory
```

**Comment by reddr** [comment]
You are right, the error message is confusing.

**Comment by eighthave** [comment]
It should say something like "no APKs found in /tmp/newdir".  I guess the hard question is whether that example is an error or not.  An error would be useful when the user is expecting there to be APKs there, but it might be annoying when running batches using the `xargs` trick, e.g.:

```console
$ find ~/data/ -mindepth 2 -maxdepth 2 -type d \
	| xargs -P8 -n1 java -jar build/LibScout.jar -o match -a lib/android-23.jar -p profiles -j json -s stats -d logs  
```

**Comment by reddr** [comment]
Fixed with 69cd7624d5f07c9837183bda210a03eb510887a0
